dpnp/tests/test_linalg.py

```diff
@@ -334,11 +333,7 @@ def test_nan(self, p):
         # while OneMKL returns nans
         if is_cuda_device() and p in [-dpnp.inf, -1, 1, dpnp.inf, "fro"]:
             pytest.skip("Different behavior on CUDA")
-        elif (
-            is_gpu_device()
-            and is_win_platform()
-            and p in [-dpnp.inf, -1, 1, dpnp.inf, "fro"]
-        ):
+        elif p in [-dpnp.inf, -1, 1, dpnp.inf, "fro"]:
```

Comment:
Do we need to check `get_intel_mkl_version` here? Since there is no need for the w/a with 2025.1 MKL.